### ISSUE TITLE
Feature Request: Define export variable

### ISSUE THREAD

**Issue by c-g-dev** [crowplexus/hscript-iris#4]
I'm trying to use Iris to drive a plugin system, where the plugins need to be both valid in normal Haxe (for development) and valid when interpreted from a text file. The main issue being that hscript/Iris returns the last variable expression, which is not something that is valid in normal Haxe.

![image](https://github.com/user-attachments/assets/d4a21cd8-6c92-4427-96c5-7885f54496a3)

Of course there are numerous ways to hack this into working but ideally there is a clean way to do it.
 
It would be beneficial to have a configuration where you could define the name of the variable that you want the interpreter to return. Like:

```haxe
 final rules:RawIrisConfig = {name: "My Script", autoRun: false, autoPreset: true, exportVariable: "export"};
var myScript:Iris = new Iris("var i = 10; var export = i;");
myScript.execute(); //returns export
```

Looking over the code, this should be trivial to implement, as you could just return the configured variable from Interp.variables.

If you approve of this change and so desire, I can implement this and raise a PR. (I'm already going to fork this and implement it for personal use anyway).

**Comment by NeeEoo** [comment]
Removing this feature would break a lot of internal systems, and haxe does do it, issue is that it only does not do it for return

**Comment by c-g-dev** [comment]
There wouldn't be any removal of features involved. It would be minimally invasive and shouldn't break any other functionality. The following implementation works without breaking the other tests:

https://github.com/crowplexus/hscript-iris/compare/master...c-g-dev:hscript-iris:master

As far as I can tell, I can't find any way to make the normal Haxe compiler accept the last expression of a module to simply be an Identity, as hscript expects. Also you can't use the return keyword from the module level.

https://try.haxe.org/#A9ec6Cfa

**Comment by NeeEoo** [comment]
Do 
```haxe
var a = {
   var i = 10;
   i;
}
```

**Comment by c-g-dev** [comment]
Yes that is the way it will need to be in the normal Haxe module. but then that won't work for hscript/Iris, which does not return a value if the last expression in the block was EVar.

```haxe
    final rules:RawIrisConfig = {name: "My Script", autoRun: false, autoPreset: true};
    var myScript:Iris = new Iris("
    var a = {
      var i = 10;
      i;
    };
    ", rules);

    trace(myScript.execute()); //null
```

**Comment by crowplexus** [comment]
this is still being discussed, for now, your fork can suffice the need to do it, I'm very used to Lua having this behaviour so I might add it at some point!
